application/app/src/main/java/com/example/buber/Services/ApplicationServiceHelper.java

```diff
@@ -0,0 +1,9 @@
+package com.example.buber.Services;
+
+import com.example.buber.Model.User;
+
+public interface ApplicationServiceHelper {
```

Comment:
What is your rational for using this interface? Not saying its wrong just curious. I like how you pass the reference to the Service class in the callback, good thinking on that one.

Reply:
"pass the reference to the Service class in the callback, good thinking on that one." To do this I needed the interface or it wouldn't work